cli_utils.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from app.database.models import Conversation
+
+
+def format_conversations(conversations: List[Conversation]):
+    formatted_conversations = []
+    for conv in conversations:
+        try:
+            formatted_conv = {
+                'user_id': conv['user_id'],
```

Comment:
isn't `conv` an instance of `Conversation`? why do you access its properties like a dict?